include/dgl/sample_utils.h

```diff
@@ -49,7 +49,8 @@ class AliasSampler: public BaseSampler<Idx, DType, replace> {
   DType accum, taken;             // accumulated likelihood
   std::vector<Idx> K;             // alias table
   std::vector<DType> U;           // probability table
-  std::vector<DType> _prob;       // category distribution
+  const DType *_prob;             // category distribution
```

Comment:
Could you please explain why using a pointer here?

Reply:
I was considering using NDArrays but right now I'm going to directly compute random walk on CSR arrays anyway.  So I'm reverting it.